editor.planx.uk/src/pages/FlowEditor/lib/analyticsProvider.tsx

```diff
@@ -28,28 +28,41 @@ const ALLOW_LIST = [
 ] as const;
 
 export type HelpClickMetadata = Record<string, string>;
-export type SelectedUrlsMetadata = Record<"selectedUrls", string[]>;
-export type BackwardsNavigationInitiatorType = "change" | "back";
+type SelectedUrlsMetadata = Record<"selectedUrls", string[]>;
+type BackwardsNavigationInitiatorType = "change" | "back";
 
-type NodeMetadata = {
+type ResultNodeMetadata = {
+  flag?: Flag;
   flagSet?: FlagSet;
   displayText?: {
     heading?: string;
     description?: string;
   };
-  flag?: Flag;
-  title?: string;
-  type?: string | null;
-  id?: string;
+};
+
+type AutoAnswerMetadata = {
   isAutoAnswered?: boolean;
 };
 
+type NodeMetadata = ResultNodeMetadata & AutoAnswerMetadata;
+
+type BackwardsTargetMetadata = {
+  id?: string;
+  type?: string | null;
+  title?: string;
```

Comment:
These have to be optional as some change events we can't access the target `nodeId` so we only track `change: {}`